packaging/fedora-25/snap-mgmt.sh

```diff
@@ -23,8 +23,8 @@ if [ "$1" = "purge" ]; then
         umount -l ${SNAP_MOUNT_DIR} || true
     fi
 
-    mounts=$(systemctl list-unit-files --full | grep '^snap[-.].*\.mount' | cut -f1 -d ' ')
-    services=$(systemctl list-unit-files --full | grep '^snap[-.].*\.service' | cut -f1 -d ' ')
+    mounts=$(systemctl list-unit-files --full | grep '^var-lib-snapd-snap[-.].*\.mount' | cut -f1 -d ' ')
+    services=$(systemctl list-unit-files --full | grep '^var-lib-snapd-snap[-.].*\.service' | cut -f1 -d ' ')
```

Comment:
Is this a generally needed change? If so, then I should pull this into my `snap-mgmt.sh` in Fedora Dist-Git.